src/main/java/dev/backend/tutor/entities/SubjectGrades/UkrMovaThemeGrades/Lexicology.java

```diff
@@ -0,0 +1,49 @@
+package dev.backend.tutor.entities.SubjectGrades.UkrMovaThemeGrades;
+
+import dev.backend.tutor.entities.SubjectGrades.UkrMovaGrades;
+import jakarta.persistence.*;
+
+
+@Entity
+@Table(name = "lexicology")
+public class Lexicology {
+    @Id
+    private String username;
+    private Integer lexicalMeaningOfAWord;
+    private Integer synonymus;
+    private Integer lexicalError;
+    private Integer phraseology;
+    @OneToOne(mappedBy = "lexicology")
+    private UkrMovaGrades ukrMovaGrades;
+
+    public String getUsername() {
+        return username;
+    }
```

Comment:
You dont need this method. Its dangerous to rename field that represenrs primary key in database. It relates other classes as well

Reply:
actually I was talking about the setter.

Reply:
Getter is fine.